plugins/wordpress-rule-exclusions-before.conf

```
@@ -982,20 +982,16 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/admin-ajax.php" \
             ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:newcontent"
 
 # Opening the plugins page
-# The plugin page contains the text "Plugins extend and expand the functionality of WordPress.
-# 953101 is matching "the function"
 SecRule REQUEST_FILENAME "@rx /wp-admin/(?:plugins|plugin-install)\.php$" \
     "id:9507971,\
     phase:4,\
```

This rule can be moved to phase 1 for better performance since it's not inspecting the respone body/headers anymore.
```suggestion
    phase:1,\
```